helm-chart/kuberay-apiserver/templates/deployment.yaml

```diff
@@ -52,6 +52,13 @@ spec:
           httpGet:
             path: /healthz
             port: http
+        env:
```

Comment:
Maybe pass to `args` instead so that we don't need to handle the complex merge logic of `env` in the future.

Reply:
done